### PR TITLE
Minimal changes to feed runtime info into the Task

### PR DESCRIPTION
**Pull request by geoffjentry**
@mcovarr note that this does not involve any of the potential changes that I was talking about the other day and is merely getting the Runtime info in the Task.

### CODE REVIEW

File: src/test/scala/cromwell/binding/ThreeStepImportNamespaceSpec.scala [hunk 1]
```diff
@@ -11,6 +11,9 @@ class ThreeStepImportNamespaceSpec extends FlatSpec with Matchers {
     |  output {
     |    File procs = stdout()
     |  }
```
Comment (scottfrazer) [inline src/test/scala/cromwell/binding/ThreeStepImportNamespaceSpec.scala]: Hmm, what do you think about creating a new test, like `RuntimeSectionSpec` where you have a minimum WDL that expresses runtime... something like

```
task test {
  command { ps }
  runtime { "docker": "ubuntu/latest" }
}

workflow wf { call test }
```

This way each test's WDL file can encompass only the thing it's testing.

**Comment by scottfrazer** [comment]
Besides that one comment about testing structure, I think it looks fine.

File: src/test/scala/cromwell/util/SampleWdl.scala [hunk 2]
```diff
@@ -34,6 +34,9 @@ object SampleWdl {
         |  output {
```
Comment (scottfrazer) [inline src/test/scala/cromwell/util/SampleWdl.scala]: does the runtime section need to be in this WDL?

Comment (geoffjentry) [inline src/test/scala/cromwell/util/SampleWdl.scala]: No, I struggled a bit to identify the right place to add it. This was an earlier attempt.

File: src/test/scala/cromwell/binding/ThreeStepImportNamespaceSpec.scala (hunk 1 above)
Comment (geoffjentry) [inline src/test/scala/cromwell/binding/ThreeStepImportNamespaceSpec.scala]: I don't disagree, I never found a place I really thought was a good one to handle the runtime (nor its own thing). However in this case the test is looking at a specific wdl and making sure everything imports correctly, so I don't see it as being a big deal either way.

That said, there should be wdl without runtime and verifying that it's empty.

Comment (geoffjentry) [inline src/test/scala/cromwell/binding/ThreeStepImportNamespaceSpec.scala]: I made a specific test for the RuntimeAttribute stuff, which is now committed

**Pull request merged**